h5py/h5p.pyx

```diff
@@ -1033,6 +1033,41 @@ cdef class PropFAID(PropInstanceID):
         return (msize, plist)
 
 
+    if ROS3 and HDF5_VERSION >= (1, 10, 6):
```

Comment:
Shouldn't need the version check here, right? `ROS3` should only be true if HDF5 is new enough to support it.